website/content/docs/concepts/domain-model/targets.mdx

```diff
@@ -81,9 +95,16 @@ TCP targets have the following additional attribute:
 ### SSH target attributes <sup>HCP/ENT</sup>
 
 SSH targets can source username/password or SSH private key credentials from Vault [credential libraries][] or static
-[credentials][]. Boundary then injects credentials into the SSH session between a client and end host. This allows users to
+[credentials][]. If you use injected credentials, Boundary injects credentials into the SSH session between a client and end host. Injected credentials allow users to
 securely connect to remote hosts using SSH, while never being in possession of a valid credential for that target host.
 
+<Note>
+
+If you set the target `type` to `ssh`, you must use brokered or injected credentials.
+You cannot reference a static .pem file.
+
+</Note>
+
```

Comment:
This note is still confusing to me. Again an SSH target requires an injected application credential. You can also have brokered credentials in addition to the injected ones, but it is not a "borkered or injected".

Reply:
Yes this isn't right based on our conversation. Something like Tim put previously with a slight edit:

<Note>

If you want to use brokered credentials or a static .pem file when connecting to a target running SSH, you must set the target type to TCP

If you want to use injected application credentials when connecting to a target running SSH, you must set the target type to SSH

 </Note>